fbw-common/src/wasm/systems/systems/src/apu/mod.rs

```diff
@@ -367,6 +368,46 @@ impl SimulationElement for AuxiliaryPowerUnitOverheadPanel {
     }
 }
 
+pub struct AirStarterUnit {
```

Comment:
Apologies for not spotting this earlier, but I think the ASU deserves to go into its own file/directory. It really doesn't have much to do with the APU.

Reply:
I wondered about that, since it didn't have much functionality. I can separate it though.